atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -204,6 +205,11 @@ def run(self):
             else:
                 self.log.error("Unknown result from add_help plugin: %s", help_result)
 
+        pulp_pull_results = self.workflow.postbuild_results.get(PulpPullPlugin.key)
+        if pulp_pull_results:
+            image_id, media_types = pulp_pull_results
```

Comment:
Since we're not using `image_id`, we can write this as `_, media_types = pulp_pull_results`.